### ISSUE TITLE
Type of CatagoryChannel holds null

### ISSUE THREAD

**Issue by NarutoInstinct** [discordjs/discord.js#2309]
**Please describe the problem you are having in as much detail as possible:**
When trying to access the type property of a CatagoryChannel, it holds null. 

**Include a reproducible code sample here, if possible:**
```js
const type = message.guild.channels.find("name", "owo").type;
console.log(type)
```
"owo" being a catagory obviously.

**Further details:**

- discord.js version: 11.3.0
- node.js version: 8.5.0
- Operating system: Windows 7 Professional
- Priority this issue should have – please be realistic and elaborate if possible:
__Medium__

I guess you're still able to check wether a channel is a catagory by checking if type is null, but this could bring up issues in people's code in the future of when the problem is fixed. Other than doing this, I wouldn't really see another way to check if a channel is a catagory, so medium seemed realistic to me.

- [x] I found this issue while running code on a __user account__
- [ ] I have also tested the issue on latest master, commit hash:

**Comment by Dev-Yukine** [comment]
this is already fixed in the 11.3 dev branch with the commit b5ff309bf985a43da4229421ac53b68f216edede, if its bother you just update the the 11.3-dev branch since it only has bugfixes at the current state and wont have any major breaking updates

**Comment by SpaceEEC** [comment]
As Yukine mentioned this has already been fixed with the linked commit.

As workarounds until 11.3.1 is released:
- assume that type `null` is `'category'`
- use an `instanceof CategoryChannel`
- install the `11.3-dev` branch from GitHub
